Formula/a/apkleaks.rb

```diff
@@ -0,0 +1,43 @@
+class Apkleaks < Formula
+  include Language::Python::Virtualenv
+
+  desc "Scanning APK file for URIs, endpoints & secrets"
+  homepage "https://github.com/dwisiswant0/apkleaks"
+  url "https://github.com/dwisiswant0/apkleaks/archive/refs/tags/v2.6.1.tar.gz"
```

Comment:
Why not install from PyPI?

Reply:
I thought it does not have pypi artifact.

Reply:
<img width="1159" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/c1856a83-6404-41e0-b7e6-aeb11b2cdd88">

Reply:
OK, then could you please raise an issue upstream, and add a comment to this formula linking to the issue?

Reply:
Sounds good, let me do it.